Furkan's suggestions

begriffs opened the pull request:
Fixes #439 
Fixes #442

sumedhpathak commented on sharding/colocation.rst:
```diff
@@ -90,12 +90,11 @@ Across all shards of the event table (Q2):
-  WHERE page_id IN (ARRAY[...page IDs from first query...]) AND
-        tenant_id = 6 AND
-        payload->>'time' >= now() - interval '1 week'
+  WHERE page_id = ANY(ARRAY[...page IDs from first query...])
```
@begriffs Have you tested this query to check that it works as expected (and makes sense in the context of this section)?

begriffs commented:
I rebased away the change to the append section.

The co-location section actually has other things in its queries that don't make sense, like a reference to `tenant_id` which doesn't exist. I'll look the section over, supply some sample data, and run the queries to get any bugs out.

begriffs commented:
I rebased away the change to the append section.

Also simplified the code in co-location section. Ran the example, adding some data in and verified that it runs.

sumedhpathak commented on sharding/colocation.rst:
```diff
@@ -90,12 +91,11 @@ Across all shards of the event table (Q2):
-  WHERE page_id IN (ARRAY[...page IDs from first query...]) AND
-        tenant_id = 6 AND
-        payload->>'time' >= now() - interval '1 week'
+  WHERE page_id IN (…page IDs from first query…)
```
This overall looks good, but I'm wondering if this section needs some rewrite now that we have broader SQL support in 7.2. For example, we could actually write this query as a subquery in 7.2, which sort of diminishes the point we're trying to make.
